### PR TITLE
add home page

### PR DESCRIPTION
**Pull request by Melaku05**
- created a hello microverse project
- implemented a main title<h1>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (Wes-Isaac) [inline .gitignore]: - [ ] Great job adding the node_modules folder in the .gitignore. However, it's one of the [requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#requirements) to also ignore a test.md file, kindly fix that.

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @Melaku05 , 

Impressive work so far! 
There are some issues that you still need to work on to go to the next project but you are almost there!

### 📝👌 Your project highlights 🔥😎

- Linters are on point ✔
- Proper use of GitHub flow 👌


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShoiraTa — APPROVED** [review]
### **STATUS: APPROVED  ✔️** 
------
Hi @Melaku05 ,  

Your project is complete! Congratulations!  🎉💯🌟 

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

### OPTIONAL

- [ ] OPTIONAL:  It looks like you forgot to update README file, please kindly update the file. with all the relevant information .This request is optional only because it was not mentioned mentioned previously. 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Melaku05** [comment]
Thank you so much for your code review.

**Pull request merged**